lib/appsignal/config.ex

```diff
@@ -214,6 +214,7 @@ defmodule Appsignal.Config do
     unless empty?(config[:files_world_accessible]) do
       System.put_env("_APPSIGNAL_FILES_WORLD_ACCESSIBLE", to_string(config[:files_world_accessible]))
     end
+    System.put_env("APP_REVISION", to_string(config[:revision]))
```

Comment:
I think we may need to have this config option be accepted as "empty string == unset" as well in the [extension (private link)](https://github.com/appsignal/appsignal-agent/blob/af2b340b003968be2423a6118bf5d488e02b928c/extension/src/worker/mod.rs#L134).

It now reports `[2018-03-13T11:11:00 (extension) #92120][Debug] Initializing extension handler with app revision` in `appsignal.log` when the value is empty, which is not correct.